lib/elixir/test/elixir/kernel/expansion_test.exs

```diff
@@ -1180,6 +1180,16 @@ defmodule Kernel.ExpansionTest do
     assert_raise CompileError, ~r"unhandled operator ->", fn ->
       expand(quote do: (foo -> bar))
     end
+
+    assert_raise CompileError,
```

Comment:
Shouldn't you also use capture_io to avoid the warning? Or we don't get a warning here?